cables.md

```diff
@@ -0,0 +1,93 @@
+# Action Cable Highlights
```

Comment:
Thanks for doing this. I read through it myself, but it's good to get insights from others.